README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
+3. Use the left and right arrow keys to position the shape.
```

Comment:
For number 2 and 3, a lot of people like using W, A, S, and D instead of the arrow keys because they're more comfortable using those instead f the arrow keys. That might want to be changed.